test/extension/handlers/topologymutation/handler_test.go

```diff
@@ -401,6 +543,8 @@ func TestHandler_GeneratePatches(t *testing.T) {
 				requestItem("3", dockerMachineTemplate, machineDeploymentVars123),
 				requestItem("4", dockerClusterTemplate, imageRepositoryVar),
 				requestItem("5", kubeadmConfigTemplate, machineDeploymentVars123),
+				requestItem("6", dockerMachinePoolTemplate, controlPlaneVarsV123WithMaxSurge),
```

Comment:
```suggestion
```
This doesn't make sense :). For DockerMachineTemplate it makes sense because DockerMachineTemplate can be used in a KubeadmControlPlane. DockerMachinePoolTemplate cannot